### PR TITLE
Added more testing relevant dependencies and a first test

### PR DESCRIPTION
**Pull request by ununseptium**
Please "zerpflückt" everything!

### CODE REVIEW

File: tests/Lappa.ORM.Tests/Generic.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿using Xunit;
+
+namespace Lappa.ORM.Tests
+{
+
+    // Class to Test Database relevant functionality of LappaORM
+    public class Databases
```
Comment (Fabi) [inline tests/Lappa.ORM.Tests/Generic.cs]: *No plural :P

File: tests/Lappa.ORM.Tests/Generic.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿using Xunit;
+
+namespace Lappa.ORM.Tests
+{
+
```
Comment (Fabi) [inline tests/Lappa.ORM.Tests/Generic.cs]: No newline

File: tests/Lappa.ORM.Tests/Generic.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    {
+        [Fact]
+        public void SQLiteDatabaseConnection()
+        {
+            var db = new Database();
+            var fabsi = db.Initialize("Data Source=:memory:", Constants.DatabaseType.SQLite);
+            Assert.True(fabsi);
+        }
+    }
+}
```
Comment (Fabi) [inline tests/Lappa.ORM.Tests/Generic.cs]: newline please

File: tests/Lappa.ORM.Tests/Generic.cs [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+{
+
+    // Class to Test Database relevant functionality of LappaORM
+    public class Databases
+    {
+        [Fact]
+        public void SQLiteDatabaseConnection()
+        {
+            var db = new Database();
+            var fabsi = db.Initialize("Data Source=:memory:", Constants.DatabaseType.SQLite);
```
Comment (Fabi) [inline tests/Lappa.ORM.Tests/Generic.cs]: fabsi?!?!?

File: tests/Lappa.ORM.Tests/Lappa.ORM.Tests.csproj [hunk 5]
```diff
@@ -8,8 +8,10 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
-    <PackageReference Include="xunit" Version="2.3.0-beta2-build3683" />
+    <PackageReference Include="xunit" Version="2.3.0-beta4-build3742" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
+    <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.0-beta4-build3742" />
```
Comment (Fabi) [inline tests/Lappa.ORM.Tests/Lappa.ORM.Tests.csproj]: Not sure if we need that one.

File: tests/Lappa.ORM.Tests/Generic.cs (hunk 4 above)
Comment (ununseptium) [reply tests/Lappa.ORM.Tests/Generic.cs]: Upsi 🍿

**Pull request merged**